feature/experimental/policy/policy_vrf_selection/ate_tests/base_vrf_selection/base_vrf_selection_test.go

```diff
@@ -377,27 +383,27 @@ func TestVrfPolicy(t *testing.T) {
 		{
 			desc:      "Match IPv4",
 			policy:    "match-ipv4",
-			passFlows: []string{"Ipv4Vlan10", "IpipVlan10", "IpipDscp46Vlan10", "IpipDscp42Vlan10"},
+			passFlows: []string{"Ipv4Vlan10", "IpipVlan10", "IpipDscp46Vlan10", "IpipDscp42Vlan10", "Ipv4Vlan20", "IpipVlan20", "IpipDscp46Vlan20", "IpipDscp42Vlan20"},
 		},
 		{
 			desc:      "Match IPinIP",
 			policy:    "match-ipip",
-			passFlows: []string{"IpipVlan10", "IpipDscp46Vlan10", "IpipDscp42Vlan10"},
+			passFlows: []string{"IpipVlan10", "IpipDscp46Vlan10", "IpipDscp42Vlan10", "IpipVlan20", "IpipDscp46Vlan20", "IpipDscp42Vlan20"},
 		},
 		{
 			desc:      "Match IPv4 and IPv6",
 			policy:    "match-ip4ip6",
-			passFlows: []string{"Ipv4Vlan10", "IpipVlan10", "IpipDscp46Vlan10", "IpipDscp42Vlan10", "Ipv6Vlan20"},
+			passFlows: []string{"Ipv4Vlan10", "IpipVlan10", "IpipDscp46Vlan10", "IpipDscp42Vlan10", "Ipv6Vlan20", "Ipv4Vlan20", "IpipVlan20", "IpipDscp46Vlan20", "IpipDscp42Vlan20", "Ipv6Vlan10"},
 		},
 		{
 			desc:      "Match IPinIP DSCP 46",
 			policy:    "match-ipip-dscp46",
-			passFlows: []string{"IpipDscp46Vlan10"},
+			passFlows: []string{"IpipDscp46Vlan10", "IpipDscp46Vlan20"},
```

Comment:
Wouldnt adding "IpipDscp46Vlan20" here fail the test?

Test requirement
    *   Matching of IPv4 to network-instance 10, and IPv6 to network-instance 20          
                Ensure that only IPv4 packets are seen at egress VLAN 10, and IPv6 at received VLAN 20.

Reply:
Latest changes pushed in. please check. Thanks.